src/yupSchema.js

```diff
@@ -469,14 +469,16 @@ export function buildYupSchema(field, config, logic) {
   } else if (inputType === supportedTypes.FIELDSET) {
     // build schema for field of a fieldset
     validators[0] = () => withBaseSchema().shape(buildFieldSetSchema(propertyFields.fields));
-  } else if (inputType === supportedTypes.FILE) {
-    validators.push(withFile);
   }
 
   if (propertyFields.required) {
     validators.push(withRequired);
   }
 
+  if (inputType === supportedTypes.FILE) {
```

Comment:
thought: Good catch. I never thought the order of validators would matter here 💡

Reply:
Indeed, took me a while to catch what was going on!